pysat/instruments/cosmic_gps.py

```diff
@@ -298,6 +307,8 @@ def download(date_array, tag, sat_id, data_path=None,
         sub_dir = 'wetPrf'
     elif tag == 'atmPrf':
```

Comment:
`atmprf`

Reply:
and done. Did a search and couldn't find any other inconsistent uses of atmprf/atmPrf